src/Games/NexusMods.Games.MountAndBlade2Bannerlord/Installers/BannerlordModInstaller.cs

```diff
@@ -0,0 +1,216 @@
+    {
+        var moduleInfoFileTuples = await GetModuleInfoFiles(libraryArchive, cancellationToken);
+        if (moduleInfoFileTuples.Count == 0) return new NotSupported(); // TODO: Will it install in root folder the content?
+
+        var launcherManager = _launcherManagerFactory.Get(loadout.Installation);
+        var store = loadout.Installation.Store;
+        var isXboxStore = store == Abstractions.GameLocators.GameStore.XboxGamePass;
+        var isNonXboxStore = !isXboxStore;
+        
+        foreach (var tuple in moduleInfoFileTuples)
+        {
+            var (moduleInfoFile, moduleInfo) = tuple;
+            var parent = moduleInfoFile.Path.Parent;
+            
+            var moduleInfoWithPath = new ModuleInfoExtendedWithMetadata(moduleInfo, ModuleProviderType.Vortex, moduleInfoFile.Path);
+            
+            var modGroup = new LoadoutItemGroup.New(transaction, out var modGroupEntityId)
+            {
+                IsGroup = true,
+                LoadoutItem = new LoadoutItem.New(transaction, modGroupEntityId)
+                {
+                    Name = moduleInfo.Name,
+                    LoadoutId = loadout,
+                    ParentId = loadoutGroup,
+                },
+            };
+            
+            var moduleInfoLoadoutItemId = Optional<EntityId>.None;
+            
+            var moduleFiles = libraryArchive.Children.Where(x => x.Path.InFolder(parent)).Select(x => x.Path.ToString()).ToArray();
+            // InstallModuleContent will only install mods if the ModuleInfoExtendedWithPath for a mod was provided
+            var installResult = launcherManager.InstallModuleContent(moduleFiles, [moduleInfoWithPath]);
+            var filesToCopy = installResult.Instructions.OfType<CopyInstallInstruction>();
+            foreach (var instruction in filesToCopy)
+            {
+                var source = instruction.Source.ToRelativePath();
+                var destination = instruction.Destination.ToRelativePath();
+                moduleInfoLoadoutItemId = AddFileFromFilesToCopy(libraryArchive, transaction, loadout,
+                    source, destination, modGroup, moduleInfoFile,
+                    moduleInfoLoadoutItemId
+                );
+            }
+            
+            var storeFilesToCopy = installResult.Instructions.OfType<CopyStoreInstallInstruction>().ToArray();
+            var hasXboxFiles = storeFilesToCopy.Any(x => x.Store == GameStore.Xbox);
+            var hasNonXboxFiles = storeFilesToCopy.Any(x => x.Store != GameStore.Xbox);
+            var usedDestinations = new HashSet<RelativePath>();
+            
+            foreach (var instruction in storeFilesToCopy)
+            {
+                // Note(sewer) Alias Xbox store with Steam store files, in case mod author
+                //      included files for only one version of the game.
+                //      For more info, see `0004-MountAndBlade2Bannerlord.md`.
```

Comment:
Is there no chance that if a mod only contains files for one store, those files are simply not going to work for other stores?
So this aliasing would be useless to negative?

Reply:
Can you rephrase the question?

In practice: 
- Almost (basically) all mods release with files for non-Xbox. 
- Xbox runs on .NET 6, while other platforms run on .NET Framework. 
- Running non-Xbox mods on Xbox is generally safe
- Running Xbox mods on non-Xbox works depending on `TargetFramework` a mod was compiled with

Reply:
And there are not API changes from the game side itself that could make one store version not work with the other version?

Reply:
Edit Note: 

In the 'unsafe' case here, if the Xbox mod doesn't work on non-Xbox, it's a Crash-To-Desktop.
If we don't copy Xbox files to non-Xbox, it's a Crash-To-Desktop.
So by default, chance of success is 0%.

Aliasing Xbox to non-Xbox however means there's a non-zero chance of a CTD, since some mods may work, depending on how they were compiled.

Reply:
You basically get an error dialog telling you the DLL is missing.
Then game usually goes poof.

Reply:
Did you discuss this feature with Aragas already? To know whether he things it would be a good idea?

Reply:
Yeah, I did.
Got the thumbs up from there.

Reply:
Perfect 👍